packages/lesswrong/components/linkPreview/FootnotePreview.tsx

```diff
@@ -167,9 +184,20 @@ const FootnotePreview = ({classes, href, id, rel, children}: {
   // it could be anything with a content-editable field in it, and that
   // information isn't wired to pass through the hover-preview system.
 
-  const onClick = useCallback(() => {
+  const onClick = useCallback((ev: React.MouseEvent) => {
     window.dispatchEvent(new CustomEvent(EXPAND_FOOTNOTES_EVENT, {detail: href}));
-  }, [href]);
+    
+    if (isRegularClick(ev) && isMobile()) {
+      setDisableHover(true);
```

Comment:
Just checking, is there a case when we'd want to set this back to `false`? I guess `isMobile()` only cares about the device and not the window size, so maybe not